packages/components/link/src/Link.stories.tsx

```diff
@@ -28,3 +29,33 @@ export const DefaultLink: Story = {
     </ThemeProvider>
   ),
 };
+DefaultLink.play = async ({
+  canvasElement,
+}: {
+  canvasElement: HTMLElement;
+}) => {
+  const canvas = within(canvasElement);
+  const link = canvas.getByRole('link');
+
+  expect(link).toHaveTextContent('Google');
+  expect(link).toHaveAccessibleName('Link to Google');
+  expect(link).toHaveStyle('color: #2860D7');
+  expect(link).toHaveStyle('font-size: 16px');
+  expect(link).toHaveStyle('background-color: rgba(0, 0, 0, 0)');
+  expect(link).toHaveStyle('position: static');
+  expect(link).toHaveStyle('width: auto');
+  expect(link).toHaveStyle('height: auto');
+  expect(link).toHaveStyle('padding: 0');
+  expect(link).toHaveStyle('clip: auto');
+  expect(link).toHaveStyle('font-weight: 400');
+  expect(link).toHaveStyle('cursor: pointer');
+  expect(link).toHaveStyle('box-sizing: content-box');
+  expect(link).toHaveStyle('opacity: 1');
+  expect(link).toHaveStyle('pointer-events: auto');
+  expect(link).toHaveAttribute('href', 'https://www.google.com');
```

Comment:
Also could be great to test target attribute with `_self` or `_blank` values

Reply:
validation added please review again